src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -29,8 +29,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
 
   if (type === CHANNEL_COMMUNITY) {
     return (
-      <View
-        style={[styles.containerImage, styles.containerDefaultImage, styles.containerBigDarkBlue]}>
+      <View style={[styles.containerImage, styles.containerDefaultImage, styles.containerDarkBlue]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style `styles.containerDarkBlue` has been used instead of `styles.containerBigDarkBlue`. If this change was intentional and the new style provides the desired appearance, then it's fine. However, if `styles.containerBigDarkBlue` was providing some specific styling related to size that is necessary for proper display, this could potentially cause a visual regression.
```diff
- <View style={[styles.containerImage, styles.containerDefaultImage, styles.containerDarkBlue]}>
+ <View style={[styles.containerImage, styles.containerDefaultImage, styles.containerBigDarkBlue]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->